src/bundle/Resources/public/js/scripts/admin.notifications.modal.js

```diff
@@ -1,89 +1,164 @@
-        const requestOptions = {
+        const notificationsTable = modal.querySelector(SELECTOR_TABLE);
+        const notificationsStatusLink = notificationsTable.dataset.notificationsCount;
+        const request = new Request(notificationsStatusLink, {
             headers: {
-                'X-CSRF-Token': token
+                'X-CSRF-Token': token,
             },
             mode: 'cors',
-            credentials: 'same-origin'
-        };
+            credentials: 'same-origin',
+        });
+
+        fetch(request)
+            .then(getJsonFromResponse)
+            .then((notificationsInfo) => {
+                setPendingNotificationCount(notificationsInfo);
+                updateModalTitleTotalInfo(notificationsInfo.total);
+            })
+            .catch(handleResponseError);
+    };
+    const updateModalTitleTotalInfo = (notificationsCount) => {
+        const modalTitle = modal.querySelector(SELECTOR_MODAL_TITLE);
+        
+        modalTitle.dataset.notificationsTotal = `(${notificationsCount})`;
+    }
+    const updatePendingNotificationsView = (notificationsInfo) => {
+        const pendingNotificationsExist = notificationsInfo.pending;
+        const userAvatar = doc.querySelector('.ez-user-menu__avatar-wrapper');
+
+        userAvatar.dataset.count = notificationsInfo.pending;
+        userAvatar.classList.toggle('n-pending-notifications', pendingNotificationsExist);
+
+        doc.querySelector('.ez-user-menu__item--notifications').dataset.count = `(${notificationsInfo.pending})`;
+    };
+    const setPendingNotificationCount = (notificationsInfo) => {
+        updatePendingNotificationsView(notificationsInfo);
 
-        fetch(modal.querySelector(SELECTOR_TABLE).dataset.notificationsCount, requestOptions)
-            .then(response => response.json())
-            .then(data => setPendingNotificationCount(data));
+        const notificationsTable = modal.querySelector(SELECTOR_TABLE);
+        const notificationsTotal = notificationsInfo.total;
+        const notificationsTotalOld = parseInt(notificationsTable.dataset.notificationsTotal, 10);
+
+        if (notificationsTotal !== notificationsTotalOld) {
+            notificationsTable.dataset.notificationsTotal = notificationsTotal;
+
+            fetchNotificationPage(currentPageLink);
+        }
+    };
+    const showNotificationPage = (pageHtml) => {
+        const modalResults = modal.querySelector(SELECTOR_MODAL_RESULTS);
+
+        modalResults.innerHTML = pageHtml;
+        toggleLoading(false);
     };
-    const getNotificationsList = () => {
-        const requestOptions = {
+    const toggleLoading = (show) => {
+        modal.classList.toggle(CLASS_MODAL_LOADING, show);
+    };
+    const fetchNotificationPage = (link) => {
+        if (!link) {
+            return;
+        }
+
+        const request = new Request(link, {
+            method: 'GET',
             headers: {
-                'X-CSRF-Token': token
```

Comment:
We don't have to send `CSRF token` for notification page but we have to fo notification count?